src/main/java/edu/cornell/kfs/concur/batch/service/ConcurEventNotificationApiService.java

```diff
@@ -2,7 +2,7 @@
 
 import edu.cornell.kfs.concur.batch.ConcurWebRequest;
 
-public interface ConcurEventNotificationV2WebserviceService {
+public interface ConcurEventNotificationApiService {
```

Comment:
since this interface defines functions to call Concur Webservices, I think Webservice should be in the interface name